calicoctl/commands/node/run.go

```diff
@@ -141,6 +142,13 @@ Options:
                            within the Docker containers that have been networked
                            by the Calico driver.
                            [default: cali]
+     --use-docker-networking-meta-data
```

Comment:
meta-data/meta data   ->  metadata

Actually, I think we should be using the term labels, not metadata.   Metadata is a more generic term, but it is labels that we are actually using.

Also, we aren't pulling selectors - the selectors are part of the policy resources used to select which endpoints the policy applies to.

I'd probably do something along the lines of:

```
--use-docker-networking-container-labels

Extract the Calico-namespaced Docker container labels
(org.projectcalico.label.*) and apply them to the container endpoints
for use with Calico policy (as opposed to using Calico profiles).  This option is only valid when using Calico
Docker networking, and when enabled will automatically disable the use of
Calico profiles to provide network isolation.
```